grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -106,6 +107,25 @@ public SystemKeyspace(InternalFactory<T> factory){
     }
 
     /**
+     * This is called when a graph is deleted via {@link GraknAdmin#delete()}.
+     * This removes the keyspace of the deleted graph from
```

Comment:
from where??? D:

Also if this removes the keyspace, shouldn't it be `deleteKeyspace`, as opposed to `keyspaceDeleted`?